docker-compose.yml

```diff
@@ -52,6 +58,10 @@ mysql-data:
 
 mysql:
   build: ./docker/images/mysql
+  entrypoint:
+    - docker-entrypoint.sh
```

Comment:
Let's keep this as-is, the set of developers affected by the backwards compatibility issue is very small, and likely of size 1.